trello_items.py

```diff
@@ -0,0 +1,58 @@
+import os
+from requests import get, put, post
+import json
+from item import ToDoItem
+
+_todo_list='5f140cd74e0efa661d85d545'
+_done_list='5f14102775f3e65db1615d81'
+
+_auth_params = {
+    'key': os.getenv('API_KEY'),
+    'token': os.getenv('USER_TOKEN')
+}
+
+def _make_item(trello_content, status):
```

Comment:
You could move this to a `@classmethod` method on the ToDoItem class.

Reply:
I considered it but I felt that the specifics of the Trello content schema should stay within the trello_items.py module. The method of how to convert a Trello content schema to a ToDoItem is not general enough to warrant putting in the class.